docs/sphinx/sections/api/apidocs/libraries/dagster-shell.rst

```diff
@@ -3,10 +3,10 @@ Shell (dagster-shell)
 
 .. currentmodule:: dagster_shell
 
-The Dagster shell library provides solid factories for executing inline shell scripts or script files.
+The Dagster shell library provides op factories for executing inline shell scripts or script files.
```

Comment:
This will affect the current docs, right? Until 0.13.0, we still want the main API docs to point to solids.

Reply:
Yea, I think there wasn't a cragified version of this. Will have to rectify that.